src/components/navBar/demoBasicUsage/index.html

```diff
@@ -0,0 +1,11 @@
+<div ng-controller="AppCtrl" ng-cloak>
+  <md-content class="md-padding">
+    <md-nav-bar selected-link-name="currentLink" nav-bar-aria-label="navigation links">
+      <md-nav-link nav-click="goto('page1')" link-name="page1">Page One</md-nav-link>
+      <md-nav-link nav-click="goto('page2')" link-name="page2">Page Two</md-nav-link>
+      <md-nav-link nav-click="goto('page3')" link-name="page3">Page Three</md-nav-link>
```

Comment:
not `ng-click` because it needs to be applied on a child element rendered by the directive, not on the directive itself.

Overriding is an intesting idea, I can probably do that.